Extension/src/main.ts

```diff
@@ -291,13 +292,12 @@ async function checkVsixCompatibility(): Promise<void> {
                 }
             }
             if (promise) {
-                promise.then(async (value) => {
-                    if (value === moreInfoButton) {
-                        await vscode.commands.executeCommand("markdown.showPreview", vscode.Uri.file(getLocalizedHtmlPath("Reinstalling the Extension.md")));
-                    } else if (value === ignoreButton) {
-                        ignoreMismatchedCompatibleVsix.Value = true;
-                    }
-                });
+                const value = await promise;
```

Comment:
I don't think we should await this promise because it's tracking a popup message.  `.then` is the right pattern for that.  We could remove `async` from the handler though and just not `await` the `executeCommand` because there's no need to do so.